src/controller/index.ts

```diff
@@ -1,7 +1,7 @@
 import jsLogger from "js-logger";
 import { noop } from "lodash";
 import {
-    NetConnection,
+    RemoteSimulator,
```

Comment:
This is a much better name for it; it also seems like this may be a good way to split the interface (remote vs. frontend simulators) in future refactors